development.in

```diff
@@ -0,0 +1,5 @@
+-e .[epub-export,mycookbook,pdf-export,spellcheck,web-import]
```

Comment:
Could you please explain what this `-e` flag stands for? I would prefer to have the full flag.

Reply:
I updated `development.in`.